src/components/encounter/ohri-encounter-form.component.tsx

```diff
@@ -505,6 +514,8 @@ export const OHRIEncounterForm: React.FC<OHRIEncounterFormProps> = ({
             uuid: visit?.uuid,
           });
       }
+
+      encounterForSubmission['orders'] = orders;
```

Comment:
This approach poses a few issues. It doesn't let you to conditionally persist orders based on other meta props like the field's visibility. Normally it should be possible to hide a field based on business logic; In such a scenario, we don't submit a hidden field's value.

The other way would be something like:

```ts
encounterForSubmission['orders'] = fields.filter(field => !field.questionOptions.isTransient && !field.questionOptions.isHidden && !field.questionOptions.isDisable).map(field => value) 
```

PS: We ought to migrate this logic to a difference place; could be some kind of processor or a form manager.